src/main/java/edu/AleksandrTreskov/mms/entity/Client.java

```diff
@@ -11,19 +10,20 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
-    @Column(name = "name", nullable = false)
+    @Column(name = "name")
```

Comment:
Entity should override toString() and hashcode() methods, please have a look at how POJO class should looks like.